templates/recentlyuploaded.html

```diff
@@ -0,0 +1,54 @@
+{% extends "template.html" %}
+
+{% block content %}
+
+    {% if articles%}
+        <table class="table table-hover table-striped">
+    <head>
+      <tr>
+          <th>subject</th>
+        <th>title</th>
+        <th>author</th>
+        <th>time</th>
+          <th>population</th>
+      </tr>
+    </head>
+    <body>
+        {% for article in articles %}
+            <td><a href="/subject/{{ article.subject_id }}">{{ article.subject.name }}</a></td>
+            <td><a href="/article/{{ article.id }}">{{ article.title }}</a></td>
+
+            <td>{{ article.time.strftime("%Y-%m-%d %H:%M") }}</td>
+
+        </tr>
+        {% endfor %}
+    </body>
+  </table>
```

Comment:
@sanoussibarkay 

- The html tag `</table>` does not have matching `<table> `.

-